templates/apps/sumo-aws-apps/sumologic-app-utils/src/sumologic.py

```diff
@@ -72,7 +72,8 @@ def post(self, method, params, headers=None, version=DEFAULT_VERSION):
 
     def put(self, method, params, headers=None, version=DEFAULT_VERSION):
         endpoint = self.get_versioned_endpoint(version)
-        r = self.session.put(endpoint + method, data=json.dumps(params), headers=headers)
+        r = self.session.put(endpoint + method, data=json.dumps(params), headers=headers,timeout=None)
```

Comment:
lets put some value for timeout may be 2 min

Reply:
Updated